system/lib/wasm_worker/wasm_worker_initialize.S

```diff
@@ -13,32 +11,37 @@
 #endif
 
 .globaltype __stack_pointer, PTR
-.globaltype __stack_end, PTR
-.globaltype __stack_base, PTR
+.globaltype __tls_size, PTR, immutable
+.functype emscripten_stack_set_limits (PTR /*__stack_base*/, PTR /*__stack_base*/) -> ()
```

Comment:
Is the second comment supposed to be `/*__stack_end*/`?

Reply:
Fixed